### PR TITLE
Doc for OSS conversion

### PR DESCRIPTION
**Pull request by justinrlee**

### CODE REVIEW

File: _posts/installation/2019-03-15-oss-to-armory.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+Here are the actual steps:
+## Back up your existing Halyard configuration
+* From your existing Halyard, run `hal backup create`.  This will create a `tar` file; make sure you store this somewhere safe and external to your Halyard machine.
+
+## Switch from OSS Spinnaker Halyard to Armory Spinnaker Halyard
+
+* If you are using the OSS Halyard Docker image, run with something like this:
+
+    ```bash
```
Comment (kevinawoo) [inline _posts/installation/2019-03-15-oss-to-armory.md]: you don't need to indent of you do backticks

File: _posts/installation/2019-03-15-oss-to-armory.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    docker run --name armory-halyard --rm \
+        -v ${PWD}/.hal:/home/spinnaker/.hal \
+        -v ${PWD}/.kube:/home/spinnaker/.kube \
+        -v ${PWD}/.secret:/home/spinnaker/.secret \
+        -it docker.io/armory/halyard-armory:1.3.2
+    ```
+
+* If you have Halyard directly installed on your machine, you can update to Armory Halyard with the Armory installer:
```
Comment (kevinawoo) [inline _posts/installation/2019-03-15-oss-to-armory.md]: Not sure if you've been recommending people to do this, but we've defaulted to just using the docker installation rather than macos. Not sure if anyone here runs macosx version

File: _posts/installation/2019-03-15-oss-to-armory.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+    ```bash
+    curl -LO https://get.armory.io/halyard/install/latest/macos/InstallArmoryHalyard.sh 
+
+    sudo bash InstallArmoryHalyard.sh --version 1.3.2
+    ```
+
+## Update the Halconfig Spinnaker version from the OSS Spinnaker version to Armory Spinnaker
+
+In Halyard, run this command to get the latest version(s) of Armory Spinnaker:
```
Comment (kevinawoo) [inline _posts/installation/2019-03-15-oss-to-armory.md]: can you add a link to the release notes here: https://docs.armory.io/release-notes/

File: _posts/installation/2019-03-15-oss-to-armory.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+categories:
+   - Installation
+description: Upgrading from OSS to Armory Spinnaker
+type: Document
+---
+
+If you are currently on OSS Spinnaker and interested in upgrading to Armory Spinnaker, you can easily upgrade if you used Halyard to install your Spinnaker cluster (you can also easily downgrade back to OSS Spinnaker)
+
+At a high level, the process to do this is as follows:
+* Back up your existing Halyard configuration
```
Comment (kevinawoo) [inline _posts/installation/2019-03-15-oss-to-armory.md]: not super important for this PR, maybe another one
but if you add `kramdown` and set https://github.com/armory/documentation/blob/master/_config.yml#L26
you can use auto generated table of contents: https://github.com/armory/documentation/blame/master/_spinnaker/configure_kayenta.md#L7-L9

File: _posts/installation/2019-03-15-oss-to-armory.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+Then, update your Halconfig to use the Armory Spinnaker version.  For example, if you're trying to use 2.2.0, run this:
+
+    ```bash
+    hal config version edit --version 2.2.0
+    ```
+
+## Deploy Armory Spinnaker, and remove any conflicting fields from your Halconfig
+
+Because Armory Spinnaker is much more heavily tested than OSS Spinnaker, certain edge capabilities in OSS Spinnaker may not be available in Armory Spinnaker.
```
Comment (kevinawoo) [inline _posts/installation/2019-03-15-oss-to-armory.md]: wdyt?
> certain edge capabilities in OSS Spinnaker **may not be** available in Armory Spinnaker yet.
